functionnal_tests/cypress/integration/app_file/add_file_error_messages_spec.js

```diff
@@ -0,0 +1,105 @@
+import { PAGES as p } from "../../support/urls_commands";
+import { SELECTORS as s } from '../../support/generic_selector_commands'
+
+const pngFile = 'artikodin.png'
+let workspaceId
+
+describe('In a workspace', function () {
+  before(() => {
+    cy.resetDB()
+    cy.setupBaseDB()
+    cy.loginAs('administrators')
+    cy.fixture('baseWorkspace').then(workspace => {
+      workspaceId = workspace.workspace_id
+    })
+  })
+
+  beforeEach(function () {
+    cy.loginAs('users')
+    cy.visitPage({ pageName: p.DASHBOARD, params: { workspaceId: workspaceId } })
+  })
+
+  afterEach(function () {
+    cy.cancelXHR()
+  })
+
+  it('should open the app file with the newly added file', function () {
+    cy.getTag({ selectorName: s.WORKSPACE_DASHBOARD })
+      .get('button[title="Upload files"]')
+      .click()
+
+    cy.dropFixtureInDropZone(pngFile, 'image/png', '.filecontent__form', 'file_exemple1.png')
+    cy.getTag({ selectorName: s.CARD_POPUP_BODY })
+      .get('[data-cy=popup__createcontent__form__button]')
+      .click()
+    cy.getTag({ selectorName: s.CONTENT_FRAME })
+      .get('.previewcomponent').should('have.length', 1)
+    cy.getTag({ selectorName: s.CONTENT_FRAME })
+      .get('.selectStatus').contains('Opened')
+    cy.url().should('include', `/ui/workspaces/${workspaceId}/contents/file/`)
```

Comment:
Use PAGES const, please.

Reply:
I tried to but I can't find a proper entry in the PAGES const. Unless asked otherwise I'll add one right away.